### PR TITLE
Accessibility check

### PR DESCRIPTION
**Pull request by sainingo**
# Report of portfolio Accessibilty
> report listing

[☑️] Page titles - No issues found
[☑️] Image text alternatives - No issues found
[☑️] text headings - No issues found
[☑️] color contrast - No issues found
[☑️] resize - No issues found
[☑️] interaction - No issues found
[☑️] moving content - site is static no moving content
[☑️] multimedia - No issues found
[☑️] the basic structure of the page - No issues found

### CODE REVIEW

File: report.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Report of portfolio Accessibilty
+> report listing
+
+[☑️]Page titles - No issues found
+[☑️]Image text alternatives - No issues found
+[☑️]text headings - No issues found
+[☑️]color contrast - No issues found
+[☑️]resize - No issues found
```
Comment (medaminedev66) [inline report.md:8]: - [ ] The accessibility of your portfolio looks great 😃, but we still have some issues when we zoom the font. Let's make sure that Text, images, and other content do not overlap.
- Your design after zooming the font :
image
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @sainingo 😃, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] The page title is not descriptive, please see the examples below and kindly fix that:
- Poor titles:
   - _Microverse project | Your portfolio._
   - _Portifolio project_
- Better page titles:
   - _Mark's portifolio_
   - _Mark full-stack webdeveloper_

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
You can also consider:
- N/V

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: report.md (hunk 1 above)
Comment (sainingo) [reply report.md:8]: Hi @medaminedev66 
Thanks for taking your time to review our code?
I would like to ask if handling text overlapping after zoom is done with javascript? or you could kindly show us how we can do that?

Comment (sainingo) [reply report.md:8]: No issues found

**Review by AnselemOdims — APPROVED** [review]
### Hi  team🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**